artifactory.py

```diff
@@ -1929,6 +1935,8 @@ def create_aql_text(*args):
             elif isinstance(arg, list):
                 arg = "({})".format(json.dumps(arg)).replace("[", "").replace("]", "")
             aql_query_text += arg
+
+        logging.debug(f"AQL query request text: {aql_query_text}")
```

Comment:
Could we move logging close to "output"? 

Right not the function is pretty "clean" (in terms of functional programming). 

For instance, if I use the function in my code it should not give me so many logs. I should be in charge when I want to log input\output for the function or not.

I'd add it to the `aql` function after the line https://github.com/devopshq/artifactory/blob/e57a9398df3c34820249573c4b9d6c0415eeec84/artifactory.py#L1920
The function is 100% is  not clean (because of requests), so it's good place to make a log message. Also it shows only AQL text from the requests that have been done.

Reply:
agreed